projects/backend/capi/byline.ts

```diff
@@ -0,0 +1,28 @@
+import { IContent, TagType } from '@guardian/capi-ts'
+import { getImageFromURL } from '../image'
+import { Image } from '../common'
+
+export const getBylineImages = (
+    article: IContent,
+): { thumbnail?: Image; cutout?: Image } | undefined => {
+    const byline = article.fields && article.fields.byline
+    if (byline == null) return undefined
+    // We could be more clever here, but multiple contributer tags wouldn't be displayed.
+    const tags = article.tags
+
+    const contributor = tags.filter(
```

Comment:
Could use `find` here which would also mean the type for `contributor` is `Tag | undefined` which is probably desirable here.